training/model_management/components/convert_model_to_mlflow/spec.yaml

```diff
@@ -28,6 +28,15 @@ inputs:
     type: string
     description: Huggingface model id (https://huggingface.co/<model_id>). A required parameter for transformers flavor. Can be provided as input here or in model_download_metadata JSON file.
     optional: true
+  
+  mlflow_flavor:
+    type: string
+    enum:
+      - transformers
+      - pyfunc
```

Comment:
Actually, we are using flavor pyfunc to store the [mlflow model](https://ml.azure.com/model/deformable_detr_twostage_refine:1/artifacts?wsid=/subscriptions/381b38e9-9840-4719-a5a0-61d9585e1e91/resourceGroups/rupaljain-rg/providers/Microsoft.MachineLearningServices/workspaces/rupal-ws-canary&tid=72f988bf-86f1-41af-91ab-2d7cd011db47).

We are using transformers for HF model since it's flavor name is [transformers](https://ml.azure.com/registries/azureml-preview/models/google-vit-base-patch16-224/version/1?tid=72f988bf-86f1-41af-91ab-2d7cd011db47#artifacts)

Please let me know if we have different convention.

Reply:
@gauravrajguru  pyfunc, implementation can be for different use case hence the suggestion. If not mmdetection may be some unique name using pyfunc for this specific usecase

Reply:
@novaturient95 - 
The name of the parameter mlflow_flavor is mis-leading in this case. As the flavours themselves are - hftransformers (not transformers) and pyfunc here. 

I think what we are trying to capture here align more with mlflow_implementation_name or something along those lines. Let's make that change in a follow up PR - in the parameter name, and align the drop down values accordingly.

Reply:
Changed the flavor name to mmlab_pyfunc